clj-refactor.el

```diff
@@ -1511,7 +1511,7 @@ front of function literals and sets."
   (interactive)
   (insert "/")
   (when (and cljr-magic-requires
-             (looking-back cljr--magic-requires-re 6))
+             (looking-back (cljr--magic-requires-re) 6))
```

Comment:
I'll clean it up after merge.
On søn. 1. feb. 2015 at 11.46 Aaron France <notifications@github.com> wrote:

> In clj-refactor.el
> <https://github.com/clojure-emacs/clj-refactor.el/pull/112#discussion_r23896589>
> :
>
> > @@ -1511,7 +1511,7 @@ front of function literals and sets."
> >    (interactive)
> >    (insert "/")
> >    (when (and cljr-magic-requires
> > -             (looking-back cljr--magic-requires-re 6))
> > +             (looking-back (cljr--magic-requires-re) 6))
>
> Do you want me to deal with this in this PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clojure-emacs/clj-refactor.el/pull/112/files#r23896589>
> .
>